.github/workflows/build.yml

```diff
@@ -0,0 +1,195 @@
+name: build
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  setup:
+    runs-on: ubuntu-18.04
```

Comment:
is 20.04 available?

Reply:
We are still using 18.04 in all benchmark environments, so I think it makes sense to have parity here until we upgrade.

Reply:
We're upgrading.

Reply:
It is available, as a "preview". Not entirely sure what that means in the context of reliability.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on